src/logos/paypal/logo.jsx

```diff
@@ -90,6 +91,7 @@ export function PPLogo({ logoColor = LOGO_COLOR.DEFAULT, ...props } : { logoColo
             { ...props }
             name={ LOGO.PP }
             alt='PP'
+            aria-label='PP'
```

Comment:
I'm thinking about the experience of a screen reader reading this. Should we spell out PP rather than using the abbreviation?